src/libraries/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -737,7 +737,7 @@ internal static void SanityCheckGenericArguments(RuntimeType[] genericArguments,
                     SR.Format(SR.Argument_NotEnoughGenArguments, genericArguments.Length, genericParameters.Length));
         }
 
-        internal static CorElementType GetUnderlyingType(RuntimeType type)
+        internal static CorElementType GetUnderlyingCorElementType(RuntimeType type)
```

Comment:
Nit: Can be changed to instance method.

Reply:
Yep. I'm just transferring code from one machine to another. Sorry, there are more changes coming.